graph/graph.go

```diff
@@ -112,15 +113,17 @@ func NewTrafficMap() TrafficMap {
 }
 
 func Id(namespace, workload, app, version, service, graphType string) (id, nodeType string) {
-	// first, check for the special-case "unknown" node
-	if UnknownWorkload == workload && UnknownApp == app && "" == service {
-		return fmt.Sprintf("source-unknown"), NodeTypeUnknown
+	// first, check for the special-case "unknown" source node
+	if UnknownNamespace == namespace && UnknownWorkload == workload && UnknownApp == app && "" == service {
+		return fmt.Sprintf("unknown_source"), NodeTypeUnknown
 	}
 
 	// It is possible that a request is made for an unknown destination. For example, an Ingress
-	// request to an unknown path. In this case everything is unknown.
-	if UnknownNamespace == namespace && UnknownWorkload == workload && UnknownApp == app && UnknownApp == service {
-		return fmt.Sprintf("dest-unknown"), NodeTypeService
+	// request to an unknown path. In this case everything the namespace may or may not be unknown.
```

Comment:
Are you missing a word in the comment here?  ...everything "_**except**_" the namespace...

Reply:
I'll fix the comment, although given that @hhovsepy is reporting that the fix failed, I may have to revisit the whole thing.